packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
             created
           example: 1587667174725
 
+    vod-asset:
+      type: object
+      table: vod_asset
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          readOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+          index: true
+        userId:
+          type: string
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+          index: true
+          description: owner of the asset
+        deleted:
+          type: boolean
+          description: Set to true when the asset is deleted
+        objectStoreId:
+          type: string
+          description: Object store ID where the asset is stored
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        name:
+          type: string
+          description:
+            Name of the asset. This is not necessarily the filename, can be a
+            custom name or title
+          example: filename.mp4
+        duration:
+          type: number
+          description: Duration of the asset in seconds (float)
+          example: 23.8328
+        aspectRatio:
+          type: string
+          description: Aspect ratio of the asset
+          example: 16:9
+        meta:
+          type: object
+          description: User input metadata associated with the asset
+          example:
+            {
+              "title": "My awesome video",
+              "description": "This is a video of my awesome life",
+              "tags": ["awesome", "life", "video"],
+            }
+        updatedAt:
+          type: string
+          description:
+            Timestamp (in milliseconds) at which the asset was last updated
+          example: 2022-01-01T02:20:00Z
+        createdAt:
+          type: string
+          description: Timestamp (in milliseconds) at which asset was created
+          example: 2022-01-01T02:20:00Z
+        format:
+          type: string
+          description: Format of the asset
+          example: mp4
+        hash:
+          type: string
+          description: Hash of the asset
+          example: sha256:hu31498hf14398
+        tracks:
+          type: array
+          description:
+            List of tracks associated with the asset when the format contemplate
+            them (e.g. mp4)
+          items:
+            type: object
```

Comment:
Need `additionalProperties: false` as well. Could have a `required: [kind]` as well, or including any other fields that are always present.